front/app/components/SiteForm/AggField.tsx

```diff
@@ -160,6 +134,20 @@ class AggField extends React.Component<AggFieldProps, AggFieldState> {
       crowdAggFilters: [],
       sorts: [],
     };
+    const preselectedUpdater = new AggFilterSiteConfigUpdater(
```

Comment:
we're using the two different updaters to inject the field dependency. this keeps the `addFilter` and `removeFilter` interface consistent between the two updaters down the component tree.